core/station/impl/src/mappers/request.rs

```diff
@@ -57,6 +59,36 @@ impl Request {
                 .collect(),
         }
     }
+
+    pub fn to_dto_with_full_info(self) -> RequestDTO {
```

Comment:
why not parameterize `fn to_dto` with a boolean if full info should be provided to avoid code duplication?

Reply:
i created a `inner_to_dto` and kept the differentiation, mostly because i find it easier to ready being expressive then having an externally exposed boolean, wdyt?

Reply:
i created a `inner_to_dto` and kept the differentiation, mostly because i find it easier to ready being expressive then having an externally exposed boolean, wdyt?